dev/cognitiveservices/data-plane/Language/examples/text/SuccessfulCustomSentimentAnalysisTaskSubmit.json

```
@@ -0,0 +1,34 @@
+{
+  "parameters": {
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "api-version": "2023-04-15-preview",
+    "Endpoint": "{Endpoint}",
+    "body": {
+      "analysisInput": {
+        "documents": [
+          {
+            "text": "The food is great. But the service is bad.",
+            "id": "1",
+            "language": "en"
+          }
+        ]
+      },
+      "tasks": [
+        {
+          "kind": "CustomSentimentAnalysis",
+          "parameters": {
+            "projectName": "{PROJECT-NAME}",
+            "deploymentName": "{DEPLOYMENT-NAME}"
+          }
+        }
+      ]
+    }
+  },
+  "responses": {
+    "202": {
+      "headers": {
+        "Operation-Location": "{Endpoint}/language/analyze-text/jobs/{jobId}?api-version=2023-04-01"
```

Should the api version be 2023-04-15-preview?

```suggestion
        "Operation-Location": "{Endpoint}/language/analyze-text/jobs/{jobId}?api-version=2023-04-15-preview"
```

---

Good catch!! Will update